src/splitInteger.test.js

```diff
@@ -3,19 +3,25 @@
 const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
-  if a value is divisible by a numberOfParts`, () => {
-
+if a value is divisible by a numberOfParts`, () => {
+  expect(splitInteger(8, 1)).toEqual([8]);
+  expect(splitInteger(6, 2)).toEqual([3, 3]);
+  expect(splitInteger(17, 4)).toEqual([4, 4, 4, 5]);
+  expect(splitInteger(32, 6)).toEqual([5, 5, 5, 5, 6, 6]);
```

Comment:
These two values are not divisible by a number of parts and the numbers are not equal in your arrays. Confuses a bit. Try to either think of the other divisible values or to remove these two lines.